eZ/Publish/Core/REST/Common/Values/Resource.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * File containing the Resource class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\REST\Common\Values;
+
+use eZ\Publish\Core\REST\Common\Value as RestValue;
+
+/**
+ * This class represents a resource.
+ */
+class Resource extends RestValue
```

Comment:
Too bad RestValue doesn't implement the base ValueObject. We could have used the same properties scheme :/